src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_IsByRefLike.cs

```diff
@@ -698,18 +686,7 @@ public class ObsoleteAttribute{}
             CompileAndVerify(text, verify: false, symbolValidator: module =>
             {
                 var type = module.ContainingAssembly.GetTypeByMetadataName("Test").GetTypeMember("S1");
-                Assert.True(type.IsByRefLikeType);
-
-                var accessibility = Accessibility.Public;
-                var attributes = type.GetAttributes();
-                Assert.Equal(1, attributes.Count());
-
-                var assemblyName = module.ContainingAssembly.Name;
-
-                var attributeType = attributes[0].AttributeClass;
-                Assert.Equal("System.Runtime.CompilerServices.IsByRefLikeAttribute", attributeType.ToDisplayString());
-                Assert.Equal(assemblyName, attributeType.ContainingAssembly.Name);
-                Assert.Equal(accessibility, attributeType.DeclaredAccessibility);
+                AssertReferencedIsByRefLikeAttributes(Accessibility.Public, type, module.ContainingAssembly.Name, hasObsolete: false);
```

Comment:
I didn't understand why `hasObsolete` is `false` here.

Reply:
My understanding is because the attribute is lacking the correct constructor, so we don't emit IL for it. @VSadov can confirm.